src/app/Plans/components/Wizard/GeneralForm.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { Form, FormGroup, TextArea, TextInput, Title } from '@patternfly/react-core';
+import SimpleSelect, { OptionWithValue } from '@app/common/components/SimpleSelect';
+import { Provider } from '@app/Providers/types';
+
+interface IGeneralFormComponent {
+  sourceProviders: Provider[];
+  targetProviders: Provider[];
+}
+
+const GeneralForm: React.FunctionComponent<IGeneralFormComponent> = ({
+  sourceProviders,
+  targetProviders,
+}: IGeneralFormComponent) => {
+  const [planName, setPlanName] = React.useState<string>('');
+  const [planDesc, setPlanDesc] = React.useState<string>('');
```

Comment:
can we change 'Desc' to 'Description' for clarity? Unless it is an extremely long name, I think it would be best to have intentionally descriptive variable names. This way our code is self documenting.